service/k8sclient/k8client.go

```diff
@@ -734,3 +751,35 @@ func (c *K8Client) volumeSpec(diskSize string) *common.VolumeSpec {
 		},
 	}
 }
+
+func (c *K8Client) CheckOperators(ctx context.Context) (*Operators, error) {
+	output, err := c.kubeCtl.Run(ctx, []string{"api-versions"}, "")
+	if err != nil {
+		return nil, errors.Wrap(err, "can't get api versions list")
+	}
+
+	apiVersions := strings.Split(string(output), "\n")
+
+	return &Operators{
+		Xtradb: c.checkOperatorStatus(apiVersions, pxcAPIVersion),
+		Psmdb:  c.checkOperatorStatus(apiVersions, psmdbAPIVersion),
```

Comment:
Looks like we have a collision in naming:
`Xtradb: c.checkOperatorStatus(apiVersions, pxcAPIVersion),`
here XtraDB vs PXC.

Reply:
Not only here, I think it's good point to discuss on backend call. Will add it to agenda